### DISCUSSION TITLE
Agenda for the Flatcar community call on Tuesday, 12th of April 9pm IST / 5:30pm CEST / 3:30pm GMT / 11:30am EDT / 8:30am PST

### DISCUSSION THREAD

**Discussion by t-lo** [flatcar/Flatcar#704] [Flatcar Office Hours]
## Agenda for the Flatcar Community Call on Tuesday, 22nd of March 2022 17:30 CET / 12:30pm EDT / 9:30am PDT

* Zoom link:  https://us06web.zoom.us/j/84336116610
  - Meeting ID: 843 3611 6610
  - Passcode: 444888
- Youtube live stream / recording: https://www.youtube.com/watch?v=dYGhoEssdoc&ab_channel=Kinvolk
- Community Calls calendar (all future calls): [link](https://calendar.google.com/calendar/u/0/embed?src=c_ii991mqrpta9en8o7ofd4v19g4@group.calendar.google.com)
  - iCal version: [link](https://calendar.google.com/calendar/ical/c_ii991mqrpta9en8o7ofd4v19g4%40group.calendar.google.com/public/basic.ics)

## Welcome
- Brief intro/check-in of the new participants in the Zoom call. Please introduce yourself and share what brings you here today.

## News
- 5.15 goes stable
- working on a community LTS based on the last stable with 5.10

## Spotlights
- Ignition, Afterburn, and Butane
- Upcoming feature: systemd-sysext for system extensions
- We have a new project board:https://github.com/orgs/flatcar-linux/projects/7/views/1

## Release planning
- Ongoing and upcoming releases (https://github.com/orgs/flatcar-linux/projects/7/views/8)

## Q&A
- Questions from community participants, answered by the Flatcar maintainers

**Comment by tormath1** [comment]
we recently merged Ignition-v3 spec in Alpha and added Flatcar support to [`coreos/butane`](https://github.com/coreos/butane) - we could quickly showcase an example. (luks for example)

**Reply by tormath1** [reply]
asked on IRC channel if some Butane developers want to join to introduce the tool and / or just say "hello" :)

**Reply by tormath1** [reply]
prepared a quick config for demo purposes and uploaded an asciinema demo in case of demo bad luck.
@bgilbert if you still interested to join, would you like to cover a butane introduction (history, etc.) ? :)

**Reply by bgilbert** [reply]
Sure, I can talk a bit.

**Comment by t-lo** [comment]
@krnowak volunteered to introduce our Flatcar/Gentoo version differences [report](https://github.com/krnowak/reports).

**Reply by t-lo** [reply]
Update: Krzesimir can't make it, let's do this in the May call.

**Comment by pothos** [comment]
While we don't have a blog post yet, I wanted to hint on the new docs for systemd-sysext and that it can be used now on Flatcar Alpha/Beta instead of Torcx or plain binaries in `/opt/bin`: https://www.flatcar.org/docs/latest/provisioning/sysext/

**Comment by t-lo** [comment]
Added an item for our new project board.
